### PR TITLE
Add known issue regarding AWS and Ubuntu for KKP

### PR DESCRIPTION
**Pull request by archups**
This adds a known issue fix https://github.com/kubermatic/machine-controller/pull/1833 to all active KKP releases to document it.

### CODE REVIEW

File: content/kubermatic/v2.23/architecture/known-issues/_index.en.md [hunk 1]
```diff
@@ -9,6 +9,37 @@ weight = 25
...
+
+```yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: KubermaticConfiguration
+metadata:
+  name: kubermatic
+  namespace: kubermatic
+spec:
+  userCluster:
+    operatingSystemManager:
```
Comment (embik) [inline content/kubermatic/v2.23/architecture/known-issues/_index.en.md:35]: this is not the right field to configure machine-controller.

**Review by embik — CHANGES_REQUESTED** [review]

File: content/kubermatic/v2.23/architecture/known-issues/_index.en.md (hunk 1 above)
Comment (archups) [reply content/kubermatic/v2.23/architecture/known-issues/_index.en.md]: fixed!

**Review by embik — APPROVED** [review]
/approve

thanks!

**Review by embik — APPROVED** [review]

**Pull request merged**